MigrationExecutorApp/Program.cs

```diff
@@ -24,8 +18,10 @@ class Program
         private static string masterkey = ConfigurationManager.AppSettings["cosmosdbkey"];
         private static string jobdb = ConfigurationManager.AppSettings["cosmosdbdb"];
         private static string jobColl = ConfigurationManager.AppSettings["cosmosdbcollection"];
+
         private static string appInsightsInstrumentationKey = ConfigurationManager.AppSettings["appinsightsinstrumentationkey"];
-        private static DocumentClient client = new DocumentClient(new Uri(endpoint), masterkey);
+
+        private static CosmosClient client = GetCustomClient("AccountEndpoint="+ endpoint + ";AccountKey="+ masterkey);
```

Comment:
You can use the endpoint and masterkey to avoid having to manually create a connection string

Reply:
Thanks, will do that.